raspberry-pi/5/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, pkgs, ... }:
+
+{
+  nixpkgs.overlays = [
+    (import ./overlay.nix)
+  ];
+
+  boot = {
+    kernelPackages = lib.mkDefault (pkgs.linuxPackagesFor pkgs.linux_rpi5);
+    initrd.availableKernelModules = [
```

Comment:
On RPi5 the latest (`nixos-unstable` latest) `nixos-generate-config` generates hardware-config with `"nvme"` module included.
Does it make sense to include it here as well?

Reply:
```suggestion
    initrd.availableKernelModules = [
       "nvme"
```

Reply:
AFAIK `nixos-generate-config` will only list `nvme` if an NVMe drive is connected? I didn't include `nvme` because it requires an addon board, though I guess there's no real harm in including it.

Reply:
If no nvme is connected, for a particular system, do we actually need to add it the initrd than?